webviz_subsurface/plugins/_parameter_response_correlation.py

```diff
@@ -36,14 +35,21 @@ class ParameterResponseCorrelation(WebvizPluginABC):
 
 * `parameter_csv`: Aggregated csvfile for input parameters with 'REAL' and 'ENSEMBLE' columns.
 * `response_csv`: Aggregated csvfile for response parameters with 'REAL' and 'ENSEMBLE' columns.
-* `ensembles`: Which ensembles in `shared_settings` to visualize.
+* `ensembles`: Which ensembles in `shared_settings` to visualize. If neither response_csv or
+               response_file is defined, the definition of ensembles implies that you want to
+               use simulation timeseries data directly from UNSMRY data. This also implies that
+               the date will be used as a response filter of type `single`.
 * `response_file`: Local (per realization) csv file for response parameters.
 * `response_filters`: Optional dictionary of responses (columns in csv file) that can be used
 as row filtering before aggregation. (See below for filter types).
-
 * `response_ignore`: Response (columns in csv) to ignore (cannot use with response_include).
 * `response_include`: Response (columns in csv) to include (cannot use with response_ignore).
-* `aggreation`: How to aggregate responses per realization. Either `sum` or `mean`.
+* `column_keys`: Simulation vectors to use as responses read directly from UNSMRY-files in the
+                 defined ensembles using fmu-ensemble. (cannot use with response_file,
```

Comment:
Remove unnecessary .

Reply:
Not following you? The white line on line 43 or?